wazo_call_logd/database/models.py

```diff
@@ -48,6 +48,7 @@ class CallLog(Base):
     )
     source_name = Column(String(255))
     source_exten = Column(String(255))
+    source_internal_name = Column(Text)
```

Comment:
Do we want to see this field only in the CSV exports? If not, I think you are missing the field in the schema

Reply:
:see_no_evil:

Reply:
Fixed